intelmq/bots/parsers/microsoft/parser_ctip.py

```diff
@@ -91,9 +91,21 @@
     "SourceIp": "source.ip",
     "SourcePort": "source.port",
     "DestinationIp": "destination.ip",
+    "DestinationIpInfo.DestinationIpAsnNumber": "source.asn",
+    "DestinationIpInfo.DestinationIpAsnOrgName": "source.as_name",
+    "DestinationIpInfo.DestinationIpCountryCode": "source.geolocation.cc",
+    "DestinationIpInfo.DestinationIpRegion": "source.geolocation.region",
+    "DestinationIpInfo.DestinationIpCity": "source.geolocation.city",
+    "DestinationIpInfo.DestinationIpPostalCode": "extra.source.geolocation.postal_code",
+    "DestinationIpInfo.DestinationIpLatitude": "source.geolocation.latitude",
+    "DestinationIpInfo.DestinationIpLongitude": "source.geolocation.longitude",
+    "DestinationIpInfo.DestinationIpMetroCode": "extra.source.geolocation.metro_code",
+    "DestinationIpInfo.DestinationIpAreaCode": "extra.source.geolocation.area_code",
+    "DestinationIpInfo.DestnationIpConnectionType": "protocol.application",
```

Comment:
I think we are mixing here source and destination. `DestinationIp` results in `destination.ip` but the corresponding asn `DestinationIpInfo.DestinationIpAsnNumber` is saved as `source.asn`.

Reply:
Indeed, this would have been a part of an ugly hack, but I'll just map destination to destination properly instead.